gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -17885,28 +17956,97 @@ properties:
     type: array
     items:
       enum:
+        - Abdomen
         - Abdominal Peritoneum
+        - Adrenal gland, NOS
+        - Adnexa
+        - Anus, NOS
+        - Appendix
+        - Bladder, NOS
+        - Bone, NOS
```

Comment:
In the ticket it is described as 'Bone' and not 'Bone, NOS'

Reply:
Changed in ticket to "Bone, NOS" by @tlicht3